modules/http4s-client/src/main/scala/io/janstenpickle/trace4cats/http4s/client/Http4sClientRequest.scala

```diff
@@ -0,0 +1,18 @@
+package io.janstenpickle.trace4cats.http4s.client
+
+import io.janstenpickle.trace4cats.http4s.common.Request_
+import io.janstenpickle.trace4cats.model.AttributeValue.{LongValue, StringValue}
+import io.janstenpickle.trace4cats.model.{AttributeValue, SemanticAttributeKeys}
+import org.http4s.Uri
+
+object Http4sClientRequest {
+  def toAttributes(req: Request_): Map[String, AttributeValue] =
+    req.uri.host.map { host =>
+      val key = host match {
```

Comment:
@janstenpickle please look. I changed it slightly. And I think we should change it in CE2.

Reply:
Oh nice change, yeah let's backport to CE2